src/taxes/components/TaxCountryDialog/styles.ts

```diff
@@ -0,0 +1,22 @@
+import { makeStyles } from "@saleor/macaw-ui";
+
+export const useStyles = makeStyles(
+  () => ({
+    inputPadding: {
+      padding: "16px 0 16px 0"
+    },
+    wrapper: {
+      overflowX: "visible",
+      padding: 0
+    },
+    scrollable: {
+      display: "flex",
+      flexDirection: "column",
+      overflowY: "scroll",
+      maxHeight: 450,
+      marginLeft: -15,
+      paddingLeft: 15
```

Comment:
I guess it's scrollbar fix; if that's the case let's drop it as it should be fixed in macaw

Reply:
It's a workaround for clipped hover shadows because you can't use overflowX visible with overflowY scroll.